compiler/daml-lf-verify/src/DA/Daml/LF/Verify/Subst.hs

```diff
@@ -7,6 +7,8 @@ module DA.Daml.LF.Verify.Subst
   , singleExprSubst
   , singleTypeSubst
   , createExprSubst
+  , emptyExprSubst
+  , concatExprSubst
```

Comment:
These two functions don't do harm but aren't strictly needed anymore. I'm fine either way, keeping them or removing them. You call. 😸

Reply:
They are still used in the `daml-lf-verification` branch, which will be merged in master at some point in the future. So I suggest just keeping them :)